ActionSheet.android.js

```diff
@@ -161,8 +159,7 @@ export default class ActionSheet extends React.Component {
     BackAndroid.addEventListener('actionSheetHardwareBackPress', this._animateOut);
   }
 
-  @autobind
-  _onSelect(index) {
+  _onSelect = (index) => {
```

Comment:
Sounds good, changing it, I also use bind personally, I was not sure what was your preferences. Also thanks for this library it works pretty well :)